bundles/org.openhab.binding.nikobus/README.md

```diff
@@ -164,20 +137,23 @@ Defines a `rollershutter-module` with address `4C6C`.
 
 ### Buttons
 
-Once an openHAB item has been configured as a Nikobus button, it will receive a status update to ON when the physical button is pressed. When an item receives the ON command from openHAB, it will send a simulated button press to the Nikobus. This means one could also define virtual buttons in openHAB with non-existing addresses (e.g., `000001`) and use those in the programming of Nikobus installation.
+Once an openHAB item has been configured as a Nikobus button, it will receive a status update to ON when the physical button is pressed.
+When an item receives the ON command from openHAB, it will send a simulated button press to the Nikobus.
+This means one could also define virtual buttons in openHAB with non-existing addresses (e.g., `000001`) and use those in the programming of Nikobus installation.
 
 To configure an item for a button in openHAB with address `28092A`, use the following format:
 
 ```
 Thing push-button 28092A
```

Comment:
@crnjan There seems to be a major flaw in the modelling. EVERY thing must at least have one configuration parameter, but it seems you have a few that do not have those at all. Note that the Thing id must not carry any semantics, but it can be defined by the user in any way he wants. Your address should thus probably be a configuration parameter.